test/uk/gov/hmrc/helptosavefrontend/connectors/HelpToSaveReminderConnectorSpec.scala

```diff
@@ -59,14 +92,25 @@ class HelpToSaveReminderConnectorSpec extends ControllerSpecWithGuiceApp with Ht
       val htsUser =
         HtsUserSchedule(nino, "user@gmail.com", "Tyrion", "Lannister", true, Seq(1), LocalDate.parse("2000-01-01"))
 
-      val response =
-        HttpResponse(200, Json.toJson(htsUser), emptyHeaders)
-      mockPost(UpdateHtsURL, Map.empty, htsUser)(Some(response))
+      val response = HttpResponse(200, Json.toJson(htsUser), emptyHeaders)
+
+      when(
+        POST,
+        UpdateHtsURL,
+        Map.empty,
```

Comment:
Use named parameters, so we can skip passing in empty values for parameters and headers

Reply:
empty values are removed